src/components/Navigation/LogoHeader.js

```diff
@@ -10,15 +10,15 @@ const Container = styled.div`
 `;
 
 const Text = styled.span`
-  font-size: ${({ theme }) => theme.header.fontSize};
-  font-weight: ${({ theme }) => theme.header.fontWeight};
+  font-size: 22px;
```

Comment:
In my experiments on side projects, I've found that it's best for the theme to only have marketing-level concerns - colors, fonts, etc.

If I want to change LogoHeader's font size, my first thought is to go to `LogoHeader.js`. Having to then backtrack to someplace in a `theme` file is a maintenance headache which is the result of (imo) an over-reaching abstraction from the theme on my part.

If we were to make this component responsive, we would just add a media query in this block. The media breakpoint could certainly be provided by some external constant to keep things consistent across the application, but why should any file but `LogoHeader.js` be concerned with how the LogoHeader reacts to different media sizes? That's my thinking at this point.

If we needed to provide a more top-level responsiveness pattern, I'd still recommend creating a new HOC-based system to provide those details to components rather than packing things into the theme.